README.md

```diff
@@ -160,3 +160,15 @@ redeemable, the callback will be invoked with an `Error` passed as the first par
 **v1.5.0 or later is required to use this method**
 
 Updates your account's display languages for the Steam store.
+
+### redeemGameKey(gameKey[, callback])
+- `gameKey` - The Steam game key you want to redeem
```

Comment:
```suggestion
### redeemSteamKey(steamKey[, callback])
- `steamKey` - The Steam activation key you want to redeem
```
or simply
```suggestion
### redeemKey(key[, callback])
- `key` - The Steam activation key you want to redeem
```

Steam keys don't only activate games, they activate any type of app. Technically, steam keys redeem steam packages.

Reply:
Or "product key" 🤔